db/migrate/20151203230940_add_date_type_to_content_view_erratum_filter_rules.rb

```diff
@@ -0,0 +1,5 @@
+class AddDateTypeToContentViewErratumFilterRules < ActiveRecord::Migration
+  def change
+    add_column :katello_content_view_erratum_filter_rules, :date_type, :string, :default => "issued", :null => false
```

Comment:
So while we need to make sure existing rules are set to 'issued', I think all new filters should be set to 'updated'.  So that means the schema defaulting to 'updated' (in case user create via hammer or api) and UI defaulting to 'updated on'.